.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -32,26 +64,33 @@ jobs:
               secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
           echo ::set-output name=vault-token::$VAULT_TOKEN
           echo ::add-mask::$VAULT_TOKEN
-      - name: Render configs for Test Runner
+      - name: Set permissions and render test configurations
         run: |
-          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-clienttests
-          chmod +x render-config.sh
-          ./render-config.sh ${{ steps.vault-token-step.outputs.vault-token }}
+          chmod +x render_config.sh
+          ./render_config.sh ${{ steps.vault-token-step.outputs.vault-token }} dev
           chmod +r rendered
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
-        with:
-          path: |
-            .gradle/caches
-            .gradle/wrapper
-          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
-          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
-      - name: Build the WSM client and run the Basic Integration test suite
+
+      # Only integration tests need a running server
```

Comment:
This workflow only runs integration tests so this comment is a little confusing

Reply:
Cleaned up comments.